content/cookbook/index.md

```diff
@@ -7,7 +7,104 @@ description:
+
+## Account Management
+
+Learn how to manage Solana accounts effectively.
+
+| Guide                                                                                     | Client        | Description                |
+| ----------------------------------------------------------------------------------------- | ------------- | -------------------------- |
+| [How to Create an Account](/content/cookbook/accounts/create-account.md)                  | web3.js       | Create new Solana accounts |
+| [How to Calculate Account Creation Cost](/content/cookbook/accounts/calculate-account.md) | web3.js       | Estimate account costs     |
+| [How to Create a PDA's Account](/content/cookbook/accounts/create-pda.md)                 | web3.js, Rust | Work with PDAs             |
+| [How to Sign with a PDA's Account](/content/cookbook/accounts/sign-with-pda.md)           | Rust          | PDA signing operations     |
+| [How to Close an Account](/content/cookbook/accounts/close-account.md)                    | Rust          | Remove accounts            |
+| [How to Get Account Balance](/content/cookbook/accounts/get-account-balance.md)           | web3.js       | Check account balances     |
+
+## Program Development
+
+Develop Solana programs with these comprehensive guides.
+
+| Guide                                                                                        | Client | Description                  |
+| -------------------------------------------------------------------------------------------- | ------ | ---------------------------- |
+| [How to Transfer SOL in a Solana Program](/content/cookbook/programs/transfer-sol.md)        | Rust   | Program-based SOL transfers  |
+| [How to Get Clock in a Program](/content/cookbook/programs/clock.md)                         | Rust   | Access program clock         |
+| [How to Change Account Size](/content/cookbook/programs/change-account-size.md)              | Rust   | Modify account sizes         |
+| [How to Do Cross Program Invocation](/content/cookbook/programs/cross-program-invocation.md) | Rust   | CPI operations               |
+| [How to Create a Program Derived Address](/content/cookbook/programs/create-pda.md)          | Rust   | Generate PDAs                |
+| [How to Read Accounts in a Program](/content/cookbook/programs/read-accounts.md)             | Rust   | Account data access          |
+| [Reading Multiple Instructions](/content/cookbook/programs/read-multiple-instructions.md)    | Rust   | Handle multiple instructions |
+| [How to Verify Accounts in a Solana Program](/content/cookbook/programs/verify-accounts.md)  | Rust   | Account verification         |
+
+## Token Operations
+
+Comprehensive guides for working with tokens on Solana.
+
+| Guide                                                                                                     | Client             | Description                    |
```

Comment:
please also remove all the `spl-token` and `metaplex` labels in the client column of all of these